### DISCUSSION TITLE
Maybe this is my first achievement ,love it

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [community/community#18392] [Other Feature Feedback, Questions, & Ideas] [answered]
Maybe this is my first achievement ,love it

**Comment by ale-lit** [comment]
love it

**Comment by zynowhere** [comment]
Isn't Arctic Code no longer available?

**Comment by sindrimt** [comment, marked as answer]
Hello good sir. Would you be so kind to mark my questing as answered. Tying to get that galaxy brain achievement, thanks ;)

**Comment by hugorbls** [comment]
Is it normal that my achievement is not displayed despite the publication I made?

**Comment by IvailoPetrov06** [comment]
yeah is normal you just need wait
